docs/build-blockchain/concepts/pricing-plans.md

```diff
@@ -1,52 +1,64 @@
 
-:::
+### How we charge?
 
-## 📌 Build v1 - API services & nodes
+Premium Tier's usage measures in *credits*. Various methods have different credit values per request. A method's credit value calculates based on a method's usage intensity and multiple factors that include using the computational, memory, storage, and network resources.
 
-We offer 5 flexible options and pricing plans to suit all budgets and
-requirements.
+PAYG model supports two ways of charging that depend on the communication protocol used:
 
-### API pricing plan
+* **Per call** — this type is used for HTTPS calls, either individual or batch ones.
+* **Per subscription** — this type is used for WebSocket communication.
 
-:::info v1 Pricing
+We charge no matter whether a request has been successful or not. In other words, a charge is taken for each request coming to our worker, regardless of a node's response.
 
-| **Feature**                    | **Intro**   | **Basic**     | **Builder**   | **Pro**               | **Growth**            | **Enterprise**          |
-| ------------------------------ | ----------- | ------------- | ------------- | --------------------- | --------------------- | ----------------------- |
-| Monthly Cost                   | **FREE**    | **$49**       | **$224**      | **$399**              | **$799**              | **$2500+**              |
-| Rate limit per second          | 20          | 200           | 600           | 1200                  | 2400                  | 500+ (scales to 10,000) |
-| Requests per day               | **170,000** | **1,000,000** | **5,000,000** | **10,000,000**        | **20,000,000**        | **Unlimited**           |
-| Requests per month in millions | 5.1m        | 30m           | 150m          | 300m                  | 600m                  | Unlimited               |
-| Archive Mode (Historical Data) | No          | No            | No            | :white_check_mark:YES | :white_check_mark:YES | :white_check_mark:YES   |
-| Terms                          | No Contract | No Contract   | No Contract   | No Contract           | No Contract           | Contract Required       |
+Here is the summary on PAYG charging:
+
+* **Correct request** (charged) — a request uses a supported method and has a valid JSON-RPC structure. 
+* **Incorrect request** (not charged) — a request has an invalid JSON-RPC structure. We can't parse such a request.
+* **Correct request + incorrect method** (charged default amount) — a request uses an unsupported method though has a valid JSON-RPC structure. We charge default amount of credits for the infrastructure usage.
+
+### Pricing calculator
+
+Below you can find our beta pricing calculator for blockchains that support standard EVM methods. For all other methods and blockchains, the default price is set to 100 credits.
```

Comment:
Suggestion: 
Use our beta pricing calculator for standard EVM methods. For all other methods and blockchains that don't support standard EVM methods, the default price is set to 100 credits.